### PR TITLE
Fix#9592: Add appveyor CI configuration file to run some small autotests

### PR DESCRIPTION
**Pull request by patrickelectric**
This PR is running in my personal configuration right now:
https://ci.appveyor.com/project/patrickelectric/ardupilot
[Build status](https://ci.appveyor.com/project/patrickelectric/ardupilot)

Fix https://github.com/ArduPilot/ardupilot/issues/9592

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+cache:
+  - $HOME/.cache -> .appveyor.yml
+
+before_build:
+  - git submodule update --init --recursive
+  - sudo apt -y install build-essential ccache g++
+  - sudo pip install -r Tools/scripts/requeriments.txt
+
+build_script:
+  - python Tools/autotest/autotest.py $CONFIGURATION -j3
```
Comment (khancyr) [inline .appveyor.yml]: It should be better to let waf decide how many proc it will use no ?

File: Tools/scripts/requeriments.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Basic modules
+argparse
+empy
+pyserial
+pexpect
+future
+lxml
+
+# Geral modules
```
Comment (khancyr) [inline Tools/scripts/requeriments.txt]: General

File: Tools/scripts/configure-ci.sh [hunk 3]
```diff
@@ -86,7 +89,4 @@ fi
 
 . ~/.profile
 
-pip install --user -U argparse empy pyserial pexpect future lxml
-pip install --user -U mavproxy
-pip install --user -U intelhex
-pip install --user -U numpy
+pip install --user -r "$SCRIPT_PATH/requeriments.txt"
```
Comment (khancyr) [inline Tools/scripts/configure-ci.sh]: requirements.txt

File: Tools/scripts/requeriments.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Basic modules
```
Comment (khancyr) [inline Tools/scripts/requeriments.txt]: rename the file to requirements.txt ?

**Comment by patrickelectric** [comment]
The install-prereqs will install unnecessary stuff. It's only necessary ccache, gcc and autotest modules.

**Comment by khancyr** [comment]
Right ! Nice ! 
Could you add the quadplane to the test matrix ?

**Comment by patrickelectric** [comment]
The quadplane test took more than 900s in my machine. I would prefer to only stay with simple and short tests.

File: Tools/scripts/configure-ci.sh [hunk 5]
```diff
@@ -86,7 +89,4 @@ fi
 
 . ~/.profile
 
-pip install --user -U argparse empy pyserial pexpect future lxml
-pip install --user -U mavproxy
-pip install --user -U intelhex
-pip install --user -U numpy
+pip install --user -r "$SCRIPT_PATH/requirements.txt"
```
Comment (peterbarker) [inline Tools/scripts/configure-ci.sh]: We've lost the `-U` flag here.

File: .appveyor.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+image: ubuntu1804
+version: '{branch}-{build}'
+
+configuration:
+  - build.APMrover2 drive.APMrover2
+  - build.ArduCopter fly.ArduCopter
```
Comment (peterbarker) [inline .appveyor.yml]: We already test this elsewhere.  If you want to keep things short you could remove this relatively long-running test.

**Comment by OXINARF** [comment]
I'm closing this because:

- I think this was from @patrickelectric's misunderstanding of Travis operation (which we discussed in Gitter and led to #9617)
- If we plan on adding another CI (and that would need some discussion), AppVeyor is really a poor choice, it only offers one job, while several others CI services offer multiple parallel jobs for free (namely the recent Azure offer)

**Pull request closed without merging**